openbas-front/src/public/components/channels/ChannelNewspaper.js

```diff
@@ -1,13 +1,15 @@
 import React, { useState } from 'react';
 import { makeStyles, useTheme } from '@mui/styles';
 import * as R from 'ramda';
-import { Typography, Grid, Card, CardHeader, CardContent, Avatar, CardMedia, Button, Dialog, DialogTitle, DialogContent, Slide } from '@mui/material';
+import { Typography, Grid, Card, CardHeader, CardContent, Avatar, CardMedia, Button, Dialog, DialogTitle, DialogContent, Slide, Chip, Tooltip } from '@mui/material';
 import { ChatBubbleOutlineOutlined, FavoriteBorderOutlined, ShareOutlined, MoreHorizOutlined } from '@mui/icons-material';
 import { useFormatter } from '../../../components/i18n';
 import Empty from '../../../components/Empty';
 import { useHelper } from '../../../store';
 import ExpandableMarkdown from '../../../components/ExpandableMarkdown';
 import { useQueryParameter } from '../../../utils/Environment';
+// eslint-disable-next-line import/extensions
+import ChannelIcon from '../../../admin/components/components/channels/ChannelIcon.js';
```

Comment:
You don't need this eslint disable and be carefull with the extension is imported (you don't need it)

Reply:
I changed it